meta-resin-common/recipes-support/hostapp-update-hooks/files/0-bootfiles

```diff
@@ -90,16 +104,31 @@ deploy() {
 # MAIN
 #
 
-export DOCKER_HOST="unix:///var/run/docker-host.sock"
-
-if [ -z "$container_id" ]; then
-	echo "[ERROR] Container ID provided as an argument is needed."
-	exit 1
-fi
+# Do a dry run for copying the boot files and figure out if we would get in an
+# out of space situation
+boot_space="$(df -B1 --output=avail $boot_mountpoint | grep -v Avail)"
```

Comment:
If this runs as part of a recovery step because the hooks of the new version failed we don't want to bailout even if it looks like there is no space for atomic swaps

Reply:
Done. Will push with next iteration.